jade/distribution.yaml

```diff
@@ -5796,6 +5796,17 @@ repositories:
       url: https://github.com/ros/rospack.git
       version: jade-devel
     status: maintained
+  rosparam_handler:
+    doc:
+      type: git
+      url: https://github.com/cbandera/rosparam_handler.git
+      version: 0.1.1
```

Comment:
I would suggest to specify a branch here to avoid having to update the entry every time you release a new version.